app/views/markets/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for @market do |f| %>
+  <%= f.label :name, "Market name:" %>
+  <%= f.text_field :name, required: :true %>
```

Comment:
:+1: I think we're about to cover data validation in the context of rails apps soon... maybe this week?